### DISCUSSION TITLE
Feature request: plot rename after completion

### DISCUSSION THREAD

**Discussion by JerkyChew899** [Chia-Network/bladebit#34] [Ideas]
I have scripts that move .plot files from a temp directory to my farmer. Currently, the Bladebit plot files are named *.plot upon creation. This doesn't play well with my scripts, as they try to move the files off the server before they're finished. Would it be possible to change the filename to .plot.tmp or something similar, only renaming them to .plot when Bladebit is done plotting?

**Comment by harold-b** [comment]
There is already a community PR for that, so it's coming: https://github.com/harold-b/bladebit/pull/21
